.github/pull_request_template.md

```diff
@@ -1,49 +1,56 @@
-<!--
-  Your description goes here: A summary of the change, including any relevant motivation and context.
+# What it does
 
-  If relevant, include a description both of the existing behavior and of the new behavior.
+<!--
+  A summary of the change, including any relevant background, motivation, and context.
+  If relevant, include a description, screenshots, and/or video of the existing and new behavior.
 -->
 
+# Links
+
 <!--
-  Other aspects to consider. uncomment and add detail for any that seem necessary:
+  Links to relevant external resources; ie, specification documents, Jira tickets, related PRs, Honeybadger errors, etc.
 -->
 
-<!-- ### Background -->
-<!-- ### Privacy -->
-<!--
-1.	Does the Project involve the collection, use or sharing of new Personal Data?
+- [spec]()
+- [Jira ticket]()
+
+# Testing story
 
-2.	Does the Project involve a new or changed use or sharing of existing Personal Data?
--->
-<!-- ### Security -->
 <!--
-Link to Jira Task where sensitive security issues are discussed privately.
+  Does your change include appropriate tests?
+  If so, please describe how the tests included in this PR are sufficient.
+  If not, please explain why this change does not need to be tested.
 -->
-<!-- ### Caching -->
-<!-- ### Deployment strategy -->
-<!-- ### Future work -->
 
-## Links
+<!-- Other aspects to consider. Delete any sections that are not relevant to your change. -->
+
+# Privacy
```

Comment:
these seem very large. I'd suggest ## or ### for all the headings in the template, so that it doesn't scroll off the screen for an empty PR description. I am open to other ideas for how to shrink it though.

Reply:
haha i was debating whether or not this was overkill. looking at it now, i think you're right -- most of the headers were ## before, so i'll change them all to that and see how it looks